### DISCUSSION TITLE
Running locally cannot hear what I am saying

### DISCUSSION THREAD

**Discussion by Lovenoreus** [microsoft/call-center-ai#379] [Q&A] [answered]
Everything is working fine when calling the container

The behaviour of calling the local app will result in no sound being heard from the caller. The app prints a new line every time a sound is heard but can register no text from the user before or after pressing 2 for English.

I have enabled all permissions since last time in the resource group so that it sticks around for when we remove services.

I am attaching the log and the permissions

(Using the latest commit 16b9118 with no code changes except configuration)

[log.txt](https://github.com/user-attachments/files/18199394/log.txt)

![Screenshot from 2024-12-19 13-58-33](https://github.com/user-attachments/assets/55b98db4-a84a-4e82-a620-e44ebb4b08ba)

**Comment by clemlesne** [comment, marked as answer]
Hello,

The logs clearly confirmed the LLM works well and it's answered normally.

1. I understand you hear nothing by phone. Is it right?
2. Did you talked during that test?
3. Did you changed any of the regions defined by default?

After, please enable debug logs, re test, and send them again if the issue persists.
